vizro-core/src/vizro/static/css/tabs.css

```
@@ -37,6 +37,10 @@ To ensure the dynamic height adjustment and prevent scrolling, the height must b
   height: 100%;
 }
 
+.tab-pane > div > .container-fluid .flex-row {
+  align-content: flex-start;
+}
+
```

Shall we rather do below? So it's always valid, not only when it's a flex-container inside a tabs?
```suggestion
.container-fluid .flex-row {
  align-content: flex-start;
}

```